pkg/util/md5/md5.go

```diff
@@ -0,0 +1,26 @@
+package version
+
+import (
+	"io/ioutil"
+
+	"k8s.io/utils/strings"
+)
+
+const md5_length = 32
+
+// ValidateFileMatchMD5 check current PlugIn MD5 matches with .md5 file
+func ValidateFileMatchMD5(fileName, md5FileName string) (bool, error) {
```

Comment:
Name: FileMatchesMD5 sounds better, because then we can use it like:
```go
if match, err := FileMatchesMD5()
...
```

If the function name is like "validate file md5" or something like that, I would expect that the first param is the filename or content, and the second is the md5 sum, not a file of md5. So, maybe (filename, md5sum string) is better?